webapp/package.json

```diff
@@ -18,63 +18,67 @@
-    "@babel/plugin-proposal-object-rest-spread": "7.5.5",
-    "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "@babel/polyfill": "7.4.4",
-    "@babel/preset-env": "7.5.5",
-    "@babel/preset-react": "7.0.0",
-    "@babel/preset-typescript": "7.3.3",
-    "@babel/runtime": "7.4.5",
-    "@types/enzyme": "3.10.3",
+    "@babel/plugin-proposal-class-properties": "7.14.5",
+    "@babel/plugin-proposal-object-rest-spread": "7.14.5",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
+    "@babel/polyfill": "7.12.1",
+    "@babel/preset-env": "7.15.4",
+    "@babel/preset-react": "7.14.5",
+    "@babel/preset-typescript": "7.15.0",
+    "@babel/runtime": "7.15.4",
+    "@types/enzyme": "3.10.18",
     "@types/jest": "27.5.1",
-    "@types/node": "12.7.0",
     "@types/react": "16.8.24",
     "@types/react-dom": "16.8.5",
-    "@types/react-intl": "3.0.0",
-    "@types/react-redux": "7.1.2",
-    "@types/react-router-dom": "4.3.4",
-    "@types/react-select": "3.0.4",
-    "@types/react-transition-group": "4.2.2",
-    "@typescript-eslint/eslint-plugin": "2.1.0",
-    "@typescript-eslint/parser": "2.34.0",
-    "babel-eslint": "10.0.2",
-    "babel-jest": "24.8.0",
+    "@types/react-redux": "7.1.17",
+    "@types/react-router-dom": "5.3.1",
+    "@types/react-select": "4.0.15",
+    "@types/react-transition-group": "4.4.7",
+    "@typescript-eslint/eslint-plugin": "5.41.0",
+    "@typescript-eslint/parser": "5.41.0",
+    "react-intl": "4.5.0",
+    "babel-eslint": "10.1.0",
+    "babel-jest": "27.4.6",
     "babel-loader": "8.2.5",
-    "babel-plugin-typescript-to-proptypes": "0.17.1",
+    "babel-plugin-typescript-to-proptypes": "1.4.2",
     "css-loader": "6.7.1",
-    "enzyme": "3.10.0",
-    "enzyme-adapter-react-16": "1.14.0",
-    "enzyme-to-json": "3.3.5",
-    "eslint": "6.0.1",
-    "eslint-import-resolver-webpack": "0.11.1",
-    "eslint-plugin-import": "2.18.0",
-    "eslint-plugin-react": "7.14.2",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.6",
+    "enzyme-to-json": "3.6.2",
+    "eslint": "8.37.0",
+    "eslint-import-resolver-webpack": "0.13.2",
+    "eslint-plugin-formatjs": "4.0.2",
+    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-import-newlines": "1.3.0",
+    "eslint-plugin-no-relative-import-paths": "1.5.0",
+    "eslint-plugin-react": "7.28.0",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "eslint-plugin-unused-imports": "2.0.0",
     "identity-obj-proxy": "3.0.0",
     "jest": "27.5.1",
-    "jest-canvas-mock": "2.1.0",
-    "jest-junit": "6.4.0",
+    "jest-canvas-mock": "2.5.2",
+    "jest-junit": "13.1.0",
     "sass": "1.60.0",
-    "sass-loader": "10.4.1",
-    "style-loader": "1.3.0",
+    "sass-loader": "12.2.0",
+    "style-loader": "3.3.1",
     "webpack": "5.76.0",
-    "webpack-cli": "4.2.0"
+    "webpack-cli": "4.9.1"
   },
   "dependencies": {
-    "core-js": "3.1.4",
+    "core-js": "3.18.3",
     "debounce-promise": "3.1.2",
     "mattermost-redux": "5.33.1",
```

Comment:
I imagine the `mattermost-redux` dependency may be holding us back from updating other dependencies?

Reply:
Not really, there are definitely some warnings coming due to mattermost-redux but they will be gone after we merge like the other PR I have created for removing the mattermost-redux one. https://github.com/mattermost/mattermost-plugin-jira/pull/1074